src/DynamoPackagesUI/Utilities/DynamoPackagesUIClient.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Greg;
+using Greg.Requests;
+using Greg.Responses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace DynamoPackagesUI.Utilities
+{
+    /// <summary>
+    /// ACG API Client, initialize rest and file client using PackageManagerClient type from Greg
+    /// </summary>
+    internal class DynamoPackagesUIClient
+    {
+        private readonly PackageManagerClient restClient;
+        private readonly PackageManagerClient restFileClient;
```

Comment:
restFileClient is not being used, I think PackageManagerClient already takes care of it. If it's not required we can remove it.